src/components/NewMessage/NewMessageUploadEditor.vue

```diff
@@ -70,8 +70,7 @@
 				:container="modalContainerId"
 				:aria-label="t('spreed', 'Post message')"
 				@upload="handleUpload"
-				@sent="handleDismiss"
-				@failure="handleDismiss" />
+				@dismiss="handleDismiss" />
```

Comment:
Following the idea "delegate posting a message to its parent when component is in dialog", this should have the dialog prop truthy and send the message from here (receive submit event) :p

Reply:
Added in fixup